src/OhmmsPETE/OhmmsVector.h

```diff
@@ -220,6 +220,13 @@ class Vector
     {
       mAllocator.deallocate(X, nAllocated);
     }
+    // when n == 0 std::allocator's return value is unspecified.
+    if(n == 0)
```

Comment:
The values set if n==0 all get overwritten by the code following the if-test

Reply:
This is why we have code review.  It's supposed to return.

Reply:
Actually now that I look at this closer you wont make it to resize_imp if n == 0.

Reply:
Some I'm reverting this.